### PR TITLE
Bugfix: Fix failing dev check in CRF

### PR DESCRIPTION
**Pull request by njayaram2**
This commit has the following changes:
- A couple of dev check files in CRF did not have the label table creation
in it. But the label table was consumed by one of the queries that led
to dev-check failure.
- Run dev check on Jenkins build instead of install check.

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/madlib-pr-build/525/

File: src/ports/postgres/modules/crf/test/crf_train_large.sql_in [hunk 1]
```diff
@@ -234,26 +234,40 @@ INSERT INTO train_new_segmenttbl VALUES
...
         ('^.+[,]$','endsWithComma'),     ('^.+er$','endsWithER'),
         ('^.+est$','endsWithEst'),       ('^.+ed$','endsWithED'),
         ('^.+s$','endsWithS'),           ('^.+ing$','endsWithIng'),
         ('^.+ly$','endsWithly'),         ('^.+-.+$','isDashSeparatedWords'),
         ('^.*@.*$','isEmailId');
-        analyze train_new_regex;
+analyze train_new_regex;
 
-        SELECT crf_train_fgen('train_new_segmenttbl', 'train_new_regex', 'crf_label', 'train_new_dictionary', 'train_new_featuretbl','train_new_featureset');
+CREATE TABLE crf_label_new (id integer,label character varying);
```
Comment (kaknikhil) [inline src/ports/postgres/modules/crf/test/crf_train_large.sql_in]: The two files `crf_test_small.sql_in` and `crf_train_large.sql_in` have different indentation. Can we make them consistent

File: src/ports/postgres/modules/crf/test/crf_test_small.sql_in [hunk 2]
```diff
@@ -90,7 +90,7 @@
         (18,'PRP$'),(19,'RB'), (20,'RBR'),  (21,'RBS'), (22,'RP'), (23,'SYM'), (24,'TO'), (25,'UH'), (26,'VB'),
         (27,'VBD'), (28,'VBG'),(29,'VBN'),  (30,'VBP'), (31,'VBZ'),(32,'WDT'), (33,'WP'), (34,'WP$'),(35,'WRB'),
         (36,'$'),   (37,'#'),  (38,''''''), (39,'``'),  (40,'('),  (41,')'),   (42,','),  (43,'.'),  (44,':');
-	analyze crf_label;
+	analyze test_crf_label;
```
Comment (kaknikhil) [inline src/ports/postgres/modules/crf/test/crf_test_small.sql_in]: Assuming that the table `crf_label` doesn't exist, why wasn't crf install check always red?

**Comment by njayaram2** [comment]
Thank you for the comments @kaknikhil . I moved out the jenkins build script to a different commit.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/madlib-pr-build/526/

**Pull request closed without merging**